### ISSUE TITLE
Color of Text in the Button is black when used in `<Header>`. Works fine in `<Content>` though

### ISSUE THREAD

**Issue by kiranraj2k** [GeekyAnts/NativeBase#517]
## react-native, react and native-base version
react-native: 0.40.0 
react: 15.4.1
native-base: 2.0.6

## Expected behaviour
The color of the text in the button is black in the header. It should ideally be white.

## Actual behaviour
![image](https://cloud.githubusercontent.com/assets/20181550/22966553/ada46c82-f388-11e6-8739-4203bc0dddd9.png)

## Steps to reproduce (code snippet or screenshot)

```
<Header>
            <Left>
              <Button transparent>
                  <Icon name='menu' />
              </Button>
            </Left>
            <Body style={{flexDirection: 'row'}}>
              <Button>
                <Text>Occasions</Text>
              </Button>
              <Button>
                <Text>Profiles</Text>
              </Button>
            </Body>
            <Right>
                <Button transparent>
                    <Icon name='add' />
                </Button>
            </Right>
        </Header>
```

## Screenshot of emulator/device
Attached above.

## Is the bug present in both ios and android or in any one of them?
Yes, I think -- I didn't test it on Android though.

**Comment by sankhadeeproy007** [comment]
You can style the text component and give it any color you want.

**Comment by sankhadeeproy007** [comment]
Fixed with 2.0.7
